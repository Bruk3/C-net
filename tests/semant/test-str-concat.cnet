int main() {
    string c = "a" + "A";
}