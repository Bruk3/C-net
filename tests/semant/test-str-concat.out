int main()
{
string c = (string : "a" + "A");
}

